Source/Diagnostics/ReducedDiags/RhoMaximum.H

```diff
@@ -0,0 +1,39 @@
+/* Copyright 2020 Neil Zaim
+ *
+ * This file is part of WarpX.
+ *
+ * License: BSD-3-Clause-LBNL
+ */
+
+#ifndef WARPX_DIAGNOSTICS_REDUCEDDIAGS_RHOMAXIMUM_H_
+#define WARPX_DIAGNOSTICS_REDUCEDDIAGS_RHOMAXIMUM_H_
+
+#include "ReducedDiags.H"
+#include "Diagnostics/ComputeDiagFunctors/ComputeDiagFunctor.H"
+
+/**
+ *  This class mainly contains a function that computes the extremas of the total charge density
```

Comment:
I would suggest this
```suggestion
 *  This class mainly contains a function that computes the extrema of the total charge density
```